gulpfile.js

```diff
@@ -14,13 +25,85 @@ export const styles = () => {
     .pipe(postcss([
       autoprefixer()
     ]))
```

Comment:
Нужно добавить сжатие CSS плагином CSSO:
```suggestion
    .pipe(postcss([
      autoprefixer(),
      csso()
    ]))
```

Reply:
Переделаны задания styles, minJS.
Удалено задание minCSS.

В задании styles я не знаю, как правильно добавить сжатие и переименование для style.css. Нужные файлы создаются, из build страницы запускаются правильно, но из-за этой вставки в код не может завершиться выполнение самого задания styles и запуститься локальный сервер с командой start.